data/toc.yaml

```diff
@@ -2034,48 +2068,14 @@ Manuals:
           title: Configure
         - path: /build/buildkit/toml-configuration/
           title: TOML configuration
-    - sectiontitle: Continuous integration
```

Comment:
Did you move GitHub Actions up because we have good support for this provider only?

Reply:
yes and we provide no first-party solutions for any other ci/cd platforms. We can add guides and examples for other platforms but then I think they could go in the guides section, as use case guides, and not manuals.  wdyt?

Reply:
per discussion in DM, I changed it back to /ci/github-actions/